### ISSUE TITLE
ValidationError while parsing IDL

### ISSUE THREAD

**Issue by eyevz** [kevinheavey/anchorpy#65]
The follow snippet fails while parsing the IDL for Jet Protocol staking program.

```python
import asyncio
from solana.rpc.async_api import AsyncClient
from solana.publickey import PublicKey
from anchorpy import Program, Provider, Wallet

async def main():
    client = AsyncClient("https://api.mainnet-beta.solana.com/")
    provider = Provider(client, Wallet.local())
    program_id = PublicKey("JPLockxtkngHkaQT5AuRYow3HyUv5qWzmhwsCPd653n")
    program = await Program.at(
        program_id, provider
    )  # ValidationError from this call
    print(program.idl.name)
    await program.close()

await main() # asyncio.run(main()) | await main()
```

**Comment by kevinheavey** [comment]
Could you share a permanent link to this version of the IDL? Since the on-chain one can change

**Comment by eyevz** [comment]
Same errors for

* Authentication program: `JPALXR88jy2fG3miuu4n3o8Jef4K2Cgc3Uypr3Y8RNX`
* Rewards program: `JET777rQuPU8BatFbhp6irc1NAbozxTheBqNo25eLQP`

**Comment by eyevz** [comment]
@kevinheavey IDLs in question available here: https://gist.github.com/eyevz/ceadc89256e57063041e6b18e23ed390

**Comment by kevinheavey** [comment]
Fixed in #66
